configure.ac

```diff
@@ -1667,6 +1681,7 @@ echo "  sanitizers    = $use_sanitizers"
 echo "  debug enabled = $enable_debug"
 echo "  gprof enabled = $enable_gprof"
 echo "  werror        = $enable_werror"
+echo "  wleveldb      = $enable_wleveldb"
```

Comment:
I'm not sure if `wleveldb` is significant enough to be part of the default `configure` output.

Reply:
Fixed.